src/defaultPane.js

```diff
@@ -21,9 +20,15 @@ module.exports = {
     return 'about '
   },
 
-  render: function (subject, dom) {
+  render: function (subject, context) {
+    var dom = context.dom
+
     var filter = function (pred, inverse) {
-      if (typeof panes.internal.predicates[pred.uri] !== 'undefined') {
+      if (
+        typeof context.session.paneRegistry.byName('internal').predicates[
+          pred.uri
+        ] !== 'undefined'
+      ) {
```

Comment:
Hmm, don't think this should be necessary. Could be because of prettier and the extra length of the line. I'll revert the uses of newlines here for now.

Reply:
Nvm, Prettier won't let me. That's the final nail in the coffin then, will revert it in a later PR ^_^